<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color:#3a3b3a;
	color:#ffffff;
	border-color:#3a3b3a;
	border-style: solid;
	border-width:2px;
}
QLineEdit{
	background-color:#ffffff;
	color:#3a3b3a;
	border-color:#3a3b3a;
	border-style:solid;;
	border-width:2px;
	height:25px;
	font:bold
}
QLineEdit::focus{
	border-style:solid;
	border-width:2px;
	border-color:#54b0cb;
}
QLabel{
	background-color:#3a3b3a;
	border:0px solid #3a3b3a;
	padding:5px
}
QPushButton{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	height:30px;
	border-color:#54b0cb;
}
QPushButton::hover{
	background-color:#448fa6;
	border-color:#448fa6;
}
QPushButton::focus{
	background-color:#448fa6;
}
QPushButton::pressed{
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	border-style:solid;
	border-color:#ffffff;
	border-width:2px;
}
QComboBox{
	background-color:#ffffff;
	border-color:#3a3b3a;
	border-style:solid;;
	border-width:2px;
	color:#3a3b3a;
	width:50px;
	height:30px;
	font:bold;
	padding-left:5px
}
QComboBox::focus{
	border-style:solid;
	border-width:2px;
	border-color:#54b0cb;
}
QComboBox QAbstractItemView {
	background-color:#ffffff;
	color:#3a3b3a;
    border: 2px solid #ffffff;
	padding-left:2px;
	padding-right:2px;
	padding-top:2px;
	padding-bottom:2px
}
QComboBox::drop-down {
   width: 24px;
   border: 2px;
}
QComboBox::down-arrow {
    image: url(&quot;/home/krsnvss/beetcount/operators_gui/res/if_chevron-left.png&quot;);
    width: 16px;
    height: 16px;
}
QComboBox::down-arrow::hover {
    image: url(&quot;/home/krsnvss/beetcount/operators_gui/res/if_chevron-down.png&quot;);
    width: 16px;
    height: 16px;
}
QTableView{
	background-color:#ffffff;
	alternate-background-color:#e2e2e2;
	color:#3a3b3a;
	selection-color:#ffffff;
	selection-background-color:#54b0cb;
	border:2px solid #ffffff;
}
QHeaderView::section{
    background-color:#448fa6;
	color:#ffffff;
	font:bold;
	height:40px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLbl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	color:#ffffff;
	font:18px bold;
	border:0px solid #3a3b3a;
}</string>
     </property>
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'12'; font-size:18px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;НОВАЯ КАРТА&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="driverEdit">
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="placeholderText">
      <string>Ф.И.О. водителя</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="truckEdit">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string> Автомобиль</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="supEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Работодатель</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="saveBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:#54b0cb;
	color:#ffffff;
	font:bold;
	height:30px;
	border-color:#54b0cb;
}
QPushButton::hover{
	background-color:#448fa6;
	border-color:#448fa6;
}
QPushButton::focus{
	background-color:#448fa6;
}
QPushButton::pressed{
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	border-style:solid;
	border-color:#ffffff;
	border-width:2px;
}</string>
     </property>
     <property name="text">
      <string>Сохранить изменения</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelBtn">
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color:#3a3b3a;
	color:#ffffff;
	font:bold;
	border:2px solid #ffffff;
}
QPushButton::hover{
	color:#448fa6;
	border:2px solid #448fa6;
}
QPushButton::pressed{
	color:#3a3b3a;
	border:2px solid #3a3b3a;
}</string>
     </property>
     <property name="text">
      <string>Закрыть без сохранения изменений</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="changeTruck">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_edit.png</normaloff>../res/if_edit.png</iconset>
   </property>
   <property name="text">
    <string>Изменить водителя</string>
   </property>
   <property name="toolTip">
    <string>Изменить водителя</string>
   </property>
  </action>
  <action name="changeEmployer">
   <property name="icon">
    <iconset>
     <normaloff>../res/if_edit.png</normaloff>../res/if_edit.png</iconset>
   </property>
   <property name="text">
    <string>Изменить пункт погрузки</string>
   </property>
   <property name="toolTip">
    <string>Изменить пункт погрузки</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
